lib/determine-basal/determine-basal.js

```diff
@@ -314,7 +314,7 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
                         basalScheduledRate_ = basalprofile[m].rate;
                         totalInsulin += accountForIncrements(basalScheduledRate_ * basDuration);
                         totalDuration -= basDuration;
-                        console.log("Dynamic ratios log: scheduled insulin added: " + accountForIncrements(basalScheduledRate_ * basDuration) + " U. Bas duration: " + basDuration.toPrecision(3) + " h. Base Rate: " + basalScheduledRate_ + " U/h" + ". Time :" + baseTime_);
+                        //console.log("Dynamic ratios log: scheduled insulin added: " + accountForIncrements(basalScheduledRate_ * basDuration) + " U. Bas duration: " + basDuration.toPrecision(3) + " h. Base Rate: " + basalScheduledRate_ + " U/h" + ". Time :" + baseTime_);
```

Comment:
Minor comment: For these logs that you're removing, I would consider just deleting them instead of commenting them out to help keep the code cleaner.

Reply:
If you want to keep them around then, my suggestion is to have a `logForDynISF` variable that's hardcoded to `false` and protects these log statements like this:

```
if (logForDynISF) { console.log("..."); }
```

where the key difference is if I were porting this and I saw a commented out print statement I wouldn't port it, but for something like the snippet above I would.

This is a minor comment though, so feel free to ignore.